Generator/Generator.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,288 @@
+// !$*UTF8*$!
```

Comment:
Is it ok to work on xcode? I'm not sure because some people not using macos might not have it. Otherwise this is fine.

Reply:
alright good to go